producer/src/secret_manager.rs

```diff
@@ -3,17 +3,20 @@ use identity_iota::{
     verification::jws::JwsAlgorithm,
 };
 use identity_stronghold::StrongholdStorage;
-use iota_sdk::client::secret::stronghold::StrongholdSecretManager;
-use iota_sdk::client::Password;
+use identity_stronghold_ext::StrongholdExtStorage;
+use iota_sdk::client::{secret::stronghold::StrongholdSecretManager, Password};
 use iota_stronghold::SnapshotPath;
-use log::{debug, info};
+use log::info;
 use std::io::{Error, ErrorKind};
 
 /// Generates or loads a Stronghold and uses the specified `KeyId` for all cryptographic operations
-#[derive(Clone)]
+#[derive(Clone, Debug)]
 pub struct SecretManager {
-    pub stronghold_storage: StrongholdStorage,
-    pub(crate) key_id: KeyId,
+    pub(crate) stronghold_storage: StrongholdStorage,
```

Comment:
In UniCore we depend on this being `pub` (here: https://github.com/impierce/ssi-agent/blob/dev/agent_shared/src/domain_linkage/mod.rs#L47-L48). Will it cause problems if we make this private again or is there an alternative for accessing the `StrongholdStorage` in order to construct a `Storage<StrongholdStorage, _>`?

Reply:
As discussed, all fields are now changed to `pub` visibility.